jina/peapods/runtimes/jinad/__init__.py

```diff
@@ -2,7 +2,7 @@
 import asyncio
 from typing import Optional
 
-from .client import PeaDaemonClient
+from .client import PodDaemonClient
```

Comment:
if there is no semantic change on `Pea/PodDaemonClient`, I believe this should stick to `PeadDaemonClient`

Reply:
`PodDaemonClient` calls `/pods` which invokes a `Pod`. Any reason we were using `PeaDaemonClient` here?